tools/driver/swift_format_main.cpp

```diff
@@ -158,11 +158,6 @@ class SwiftFormatInvocation {
       InputFilenames.push_back(A->getValue());
     }
 
-    if (InputFilenames.empty()) {
-      Diags.diagnose(SourceLoc(), diag::error_mode_requires_an_input_file);
```

Comment:
I think so.  It simplifies the integration and matches the behaviour for clang-format.